src/styles/general.scss

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: TTNorms;
+  font-display: swap;
+  font-weight: 300;
+  src: url("/src/fonts/TTNorms-Light.woff2") format('woff2');
+}
+
+@font-face {
+  font-family: TTNorms;
+  font-display: swap;
+  font-weight: 400;
+  src: url("/src/fonts/TTNorms-Regular.woff2") format('woff2');
+}
```

Comment:
All font faces to fonts file

structure--

mixins/
--typography.scss
--flex.scss
--media.scss
base/
--fonts.scss
--general.scss - container, HTML, body styles
--variables.scss colors, gradients
--reset.scss
components/...
pages